src/langcheck/augment/_common/_jailbreak_template.py

```diff
@@ -31,6 +33,7 @@ def jailbreak_template_common(
             of templates.
         seed: The seed for the random number generator. You can fix the seed to
             deterministically select the same templates.
+        custom_templates: A list of tuples of names and paths to custom Jinja2 templates.
```

Comment:
Could you add a bit more explanation about the expected jinja template format? As you see in the line below, our existing jinja templates expect `{{ input_query }}` as the placeholder of the user-provided prompt.

https://github.com/matchcase/langcheck/blob/9747eb85ff895d364fd7db41120ed2f054510f2b/src/langcheck/metrics/prompts/en/jailbreak_templates/basic.j2#L3